zanata-common-api/src/main/java/org/zanata/rest/dto/GlossaryEntry.java

```diff
@@ -81,12 +81,12 @@ public void setSrcLang(LocaleId srcLang) {
 
     @XmlElement(name = "source-reference", required = false,
             namespace = Namespaces.ZANATA_OLD)
-    public String getSourcereference() {
-        return sourcereference;
+    public String getSourceReference() {
+        return sourceReference;
```

Comment:
This will make it not backward compatible (for JSON serialization). It's probably ok since internally only maven client uses glossary API and it's using xml marshalling. But we may want to mention this in release notes. Or create a ```@Deprecate``` method using the old method name.